Features/AdminComponentPreview/README.md

```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Enables components to be previewed in multiple scenarios:
+
+## Front end
+
+Add a full width layout screenshot of a component to its component folder to show a preview in the Wordpress backend.
+
+When logged in, you can overlay the screenshots with opacity in the frontend for easy pixel perfect development by clicking the 'Component Preview' button in the WP Admin Bar, drag the images around and move each image with the arrow keys.
```

Comment:
Can make it less passive:

`When logged in, click the 'Component Preview' button on the WordPress admin bar to overlay a preview image (with reduced opacity). For easy pixel perfect development, drag the images around or move them with the arrow keys.